src/popup/pages/Invite.vue

```diff
@@ -143,7 +143,7 @@ export default defineComponent({
       }
 
       addInvite(Buffer.from(secretKey, 'hex').slice(0, 32));
-      formModel.value.amount = '';
+      resetField();
```

Comment:
Why not just setting the `formModel.amount` to null / `''` ?

Reply:
Because the field is dirty and will show an error. Resetting it cleans it